src/rules/no-large-snapshots.ts

```diff
@@ -1,20 +1,35 @@
+import {
+  AST_NODE_TYPES,
+  TSESLint,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
 import { isAbsolute } from 'path';
-import { getDocsUrl, getStringValue } from './util';
+import {
+  createRule,
+  getAccessorValue,
+  isExpectMember,
+  isSupportedAccessor,
```

Comment:
thoughts on landing these new guards on master along with the refactor of this rule?

Reply:
done via #376